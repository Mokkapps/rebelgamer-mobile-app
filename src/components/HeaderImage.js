// @flow

import { Image, StyleSheet, Platform } from 'react-native';
import React from 'react';

const headerImage = require('../../assets/header.png');

type Props = {};
type State = {};

const styles = StyleSheet.create({
  headerImage: {
    flex: 1,
    marginLeft: Platform.OS === 'android' ? 50 : 0,
    marginTop: 5,
    height: 40,
    resizeMode: 'contain'
  }
});

class HeaderImage extends React.Component<Props, State> {
  render() {
    return <Image style={styles.headerImage} source={headerImage} />;
  }
}

export default HeaderImage;
