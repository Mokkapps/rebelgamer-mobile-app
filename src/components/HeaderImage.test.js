import React from 'react';
import renderer from 'react-test-renderer';

import HeaderImage from './HeaderImage';

it('renders correctly', () => {
  const tree = renderer.create(<HeaderImage />).toJSON();
  expect(tree).toMatchSnapshot();
});
