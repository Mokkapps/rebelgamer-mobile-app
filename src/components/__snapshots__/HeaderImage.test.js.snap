// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "flex": 1,
      "height": 40,
      "marginLeft": 0,
      "marginTop": 5,
      "resizeMode": "contain",
    }
  }
/>
`;
